packages/theming/atlas/src/themesource/atlas_core/web/main.scss

```diff
@@ -1,90 +1,138 @@
-// Utilities
-@import "core/_legacy/bootstrap/bootstrap";
-@import "core/_legacy/bootstrap/bootstrap-rtl";
-@import "core/_legacy/mxui";
-
-//================================== CORE ==================================\\
+// Main utilities
+@import "mixins/include-widget-styles";
 
-// Default variables
+// Variables
 @import "variables";
+@import "core-variables";
 @import "../../../theme/web/custom-variables";
 
+// Utilities
+@include include-styles("core/_legacy/bootstrap/bootstrap", $exclude-bootstrap);
+@include include-styles("core/_legacy/bootstrap/bootstrap-rtl", $exclude-bootstrap-rtl);
+@include include-styles("core/_legacy/mxui", $exclude-mxui);
+
+//================================== CORE ==================================\\
+
 // Base
 @import "core/base/mixins/animations";
```

Comment:
As everything is conditional now in `main.scss` and imports are done within the files themselves because of `meta.load-css` mixin, we can remove this. Good catch!